tools/ci_build/build.py

```diff
@@ -311,6 +311,7 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
 
     if args.use_trt:
         cmake_args += ["-DTENSORRT_ROOT=%s" % args.trt_path]
+        cmake_args += ["-DCUDA_HOME=%s" % args.cuda_home]
 
```

Comment:
use_cuda will be set when use_trt is true, so shouldn't any CUDA specific additions be done under args.use_cuda instead of here?

Reply:
cuda_home is set in setup_cuda_vars, but it's not passed to the cmake. I will add -Donnxruntime_CUDA_HOME instead of a variable specifically for tensorrt.